### PR TITLE
https://issues.redhat.com/browse/ACM-3236

### PR DESCRIPTION
**Pull request by oafischer**
https://issues.redhat.com/browse/ACM-3236

### CODE REVIEW

File: multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc [hunk 1]
```diff
@@ -62,19 +62,37 @@ The managed cluster administrator must create a new valid `kubeconfig` file for
 
 After creating a new `kubeconfig`, complete the following steps to update the new `kubeconfig` for your managed cluster:
 
-. Run the following command to specify your managed cluster name by replacing `your-managed-cluster-name` with the name of your managed cluster:
+. Run the following command tov set your `kubeconfig` file path. Replace `<path-to-kubeconfig>` with the path to your new `kubeconfig` file:
```
Comment (cadawson) [inline multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]:
```suggestion
. Run the following command to set your `kubeconfig` file path. Replace `<path-to-kubeconfig>` with the path to your new `kubeconfig` file:
```

**Review by cadawson — APPROVED** [review]
One minor change. The rest looks good. Thanks!

File: multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc [hunk 2]
```diff
@@ -62,19 +62,37 @@ The managed cluster administrator must create a new valid `kubeconfig` file for
 
 After creating a new `kubeconfig`, complete the following steps to update the new `kubeconfig` for your managed cluster:
 
-. Run the following command to specify your managed cluster name by replacing `your-managed-cluster-name` with the name of your managed cluster:
+. Run the following command to set your `kubeconfig` file path. Replace `<path-to-kubeconfig>` with the path to your new `kubeconfig` file:
 +
 ----
-cluster_name=<your-managed-cluster-name>
+kubeconfig_file=<path-to-kubeconfig>
```
Comment (skeeey) [inline multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]: we still need the variable `cluster_name`, so here we need set both of `cluster_name` and `kubeconfig_file`, thanks

File: multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc [hunk 3]
```diff
@@ -62,19 +62,37 @@ The managed cluster administrator must create a new valid `kubeconfig` file for
...
+kubeconfig=$(cat ${kubeconfig_file} | base64 -w0)
+----
++
+*Note:* On macOS, run the following command instead:
++
+----
+(cat ${kubeconfig_file} | base64)
+----
+
+. Run the following command to define the `kubeconfig` path:
```
Comment (skeeey) [inline multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]: path -> json patch, this patch is a json string for `oc patch` command

File: multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc [hunk 4]
```diff
@@ -62,19 +62,37 @@ The managed cluster administrator must create a new valid `kubeconfig` file for
...
+. Run the following command to encode your new `kubeconfig`:
++
+----
+kubeconfig=$(cat ${kubeconfig_file} | base64 -w0)
+----
++
+*Note:* On macOS, run the following command instead:
++
+----
+(cat ${kubeconfig_file} | base64)
```
Comment (skeeey) [inline multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]: I think we may need a whole command
kubeconfig=$(cat ${kubeconfig_file} | base64)

**Comment by skeeey** [comment]
@oafischer I have a few comments, PTAL, thank you

**Comment by oafischer** [comment]
Thanks @skeeey, let me know if it looks good now!

File: multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc [hunk 5]
```diff
@@ -62,19 +62,38 @@ The managed cluster administrator must create a new valid `kubeconfig` file for
...
+*Note:* On macOS, run the following command instead:
++
+----
+kubeconfig=$(cat ${kubeconfig_file} | base64)
+----
+
+. Run the following command to define the `kubeconfig` json patch:
++
+----
+kubeconfig_patch="\\{\"op\":\"replace\", \"path\":\"/data/kubeconfig\", \"value\":\"${kubeconfig}\"}"
```
Comment (skeeey) [inline multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]: @oafischer this should be
```
kubeconfig_patch="[{\"op\":\"replace\", \"path\":\"/data/kubeconfig\", \"value\":\"${kubeconfig}\"}]"
```

Comment (oafischer) [reply multicluster_engine/support_troubleshooting/trouble_cluster_reimport_fails_mce.adoc]: Fixed, thanks!

**Review by skeeey — APPROVED** [review]

**Pull request merged**